blackbox/handlers/storage/s3.py

```diff
@@ -101,13 +101,15 @@ def rotate(self, database_id: str) -> None:
         if Blackbox.retention_days:
             retention_days = Blackbox.retention_days
 
+        # Get files object from remote S3 bucket.
+        remote_objects = self.client.list_objects_v2(Bucket=self.bucket).get("Contents")
+
+        # Filter their names with only this kind of database.
+        this_type_backups = [item for item in remote_objects
```

Comment:
This variable name is a bit wonky to me.

Maybe something like `relevant_backups`.